nbbuild/antsrc/org/netbeans/nbbuild/CustomJavac.java

```diff
@@ -66,21 +66,30 @@ public void setIncludes(String includes) {
 
     @Override
     public void execute() throws BuildException {
-        String tgr = getTarget();
```

Comment:
Why is this necessary?  The original implementation moved target to release - why not keep that behavior?

Reply:
it still does that if release isn't set, scroll down.

Reply:
No - why setting release? It is not necessary. The original change ensured, that target is transferred to release, unless the bootclasspath is set.

Reply:
https://github.com/apache/netbeans/pull/3278#discussion_r760538319

Reply:
+ https://github.com/apache/netbeans/pull/3278#discussion_r760775688

Reply:
Not an answer

Reply:
this change ensures that javac and nb-javac do not diverge configuration wise, as mentioned in the linked conversation.

Reply:
what is the combination of attributes/JDKs which should be supported but isn't after that change?